data/src/main/java/ch/srg/dataProvider/integrationlayer/data/remote/Program.kt

```diff
@@ -21,10 +21,10 @@ data class Program(
     val endTime: Date,
     override val lead: String? = null,
     override val description: String? = null,
-    override val imageUrl: ImageUrl,
-    override val imageFocalPoint: FocalPoint? = null,
-    override val imageTitle: String? = null,
-    override val imageCopyright: String? = null,
+    val imageUrl: ImageUrl? = null,
+    val imageFocalPoint: FocalPoint? = null,
+    val imageTitle: String? = null,
+    val imageCopyright: String? = null,
```

Comment:
Lot of duplicated code, maybe we don't have other choice. Don't know how to do it better.